src/components/Pill.tsx

```diff
@@ -27,8 +27,19 @@ const Secondary = styled.div`
   font-size: 0.75rem;
   border-radius: 0.25rem;
 `
+export interface IProps {
+  children: JSX.Element
```

Comment:
No need to specify `children` here since it is defined by `React.FC`

Reply:
Learned something new. Thanks